alf/environments/suite_carla.py

```diff
@@ -1027,7 +1027,7 @@ def __init__(self,
         Args:
             batch_size (int): the number of learning vehicles.
             map_name (str): the name of the map (e.g. "Town01")
-            vehicle_filter (str): the filter for getting vehicle blueprints.
+            vehicle_filter (str): the filter for getting ego vehicle blueprints.
```

Comment:
I think 'ego' is a little confusing here. Might just explain that this filter is used for vehicles that will be trained to drive. And the other vehicles will use a catch-all filter.

Reply:
Updated.